peps/pep-0738.rst

```diff
@@ -300,6 +300,8 @@ namedtuple containing the following:
 * ``device`` - `device name
   <https://developer.android.com/reference/android/os/Build#DEVICE>`__ of the
   device, as a string (e.g. ``"panther"``)
+* ``is_simulator`` - ``True`` if the device is an emulator; ``False`` if it’s a
+  physical device.
```

Comment:
Should these two modifications (min_ap_level removed; is_simulator added) be listed as a note in the "Rejected Ideas" section? My understanding is that the final PEP should reflect the historical revisions, so there's documentation of the reasoning behind any changes that have been made.

Reply:
Done.